test/browser/isolation.js

```diff
@@ -714,5 +714,188 @@ describe('isolation', function () {
       done()
     }, 200)
     dispose = run();
+  });
+
+  it('should allow an isolated child to receive events when it is used as ' +
+    'the vTree of an isolated parent component', (done) => {
```

Comment:
I'd rename the title to "when the parent exports the child's DOM directly without wrapping".